client/me/help/help-contact-form/index.jsx

```diff
@@ -206,6 +206,19 @@ export const HelpContactForm = React.createClass( {
 	},
 
 	/**
+	 * Tracks clicks on the questions returned by Sibyl.
+	 * @param {object} event Click event
+	 * @param {object} helpLink Help link, including id for tracking
+	 */
+	trackSibylClick( event, helpLink ) {
+		// bump the clicks stat for the question in MC
+		analytics.mc.bumpStat( 'sibyl_question_clicks', helpLink.id );
+		// track that the user clicked the question, so we can see how
+		// effective questions are at preventing unnessecary chat sessions
+		analytics.tracks.recordEvent( 'calypso_sibyl_question_click' );
```

Comment:
When possible, we like to avoid using a direct call to the analytics library since this makes testing components much more difficult. (It assumes window is available).

See [redux analytics middleware](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics) for more documentation.

In this case we can dispatch a redux action that will record these analytics events. If we have a related redux event we're already firing like `COMMENTS_ADD`, we can decorate the action with analytics meta with our intent, so only one action is fired, and the analytics middleware handles the side-effect of calling the analytics libs for you.

In this case the code might look like:
```jsx
const trackSibylClick = ( helpLinkId ) => composeAnalytics( 
        recordTracksEvent( 'calypso_sibyl_question_click' ),
        bumpStat( 'calypso_sibyl_question_click', helpLinkId )
);
const mapDispatchToProps = {
    //...
    trackSibylClick,
};
```
We'd then pass through `this.props.trackSibylClick` to the onClick method.